scripts/plotting/tob_wgs_snp_chip_plot_pca/plot_pca_tob_wgs_snp_chip_datasets.py

```diff
@@ -54,18 +54,10 @@ def query():  # pylint: disable=too-many-locals
         plot_filename = output_path('pc' + str(pc2) + '.png', 'web')
         with hl.hadoop_open(plot_filename, 'wb') as f:
             get_screenshot_as_png(p).save(f, format='PNG')
-        plot_filename_html = 'pc' + str(pc2) + '.html'
-        output_file(plot_filename_html)
-        save(p)
-        subprocess.run(
-            [
-                'gsutil',
-                'cp',
-                plot_filename_html,
-                output_path(plot_filename_html, 'web'),
-            ],
-            check=False,
-        )
+        html = file_html(p, CDN, 'my plot')
+        plot_filename_html = output_path('pc' + str(pc2) + '.html', 'web')
```

Comment:
Just a tip: f-strings are quite handy -- might also make some other string concatenation above more readable.

```suggestion
        plot_filename_html = output_path(f'pc{pc2}.html', 'web')
```

Reply:
great suggestion, thanks Leo!